api/src/migrations/1543494409127-change-translation-value-type.ts

```
@@ -1,29 +1,75 @@
     switch (config.db.default.type) {
-      case 'postgres':
+      case DbType.POSTGRES:
         await queryRunner.query(`ALTER TABLE translation ALTER COLUMN value SET NOT NULL;`);
         await queryRunner.query(`ALTER TABLE translation ALTER COLUMN value SET DATA TYPE text;`);
         break;
-      case 'mysql':
+      case DbType.MYSQL:
         await queryRunner.query('ALTER TABLE `translation` MODIFY COLUMN `value` text NOT NULL');
         break;
+      case DbType.BETTER_SQLITE3:
+        const tableInfo = await queryRunner.query(`PRAGMA table_info(translation)`);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)